File: day01/jl/part2.jl
ret = read("input.txt",String)

#print(ret)

thefloor=0
step = 0
for i in 1:length(ret)
	if ret[i]=='('
		global thefloor+=1
	elseif ret[i]==')'
		global thefloor-=1
	end
	global step+=1
	if thefloor==-1
		println("Final answer: $step")
		break
	end
end

File: day01/py/part2.py
#Read data 
text_file = open("input.txt", "r")
lines = text_file.readlines()

mystring=lines[0]
thefloor=0
step=0
for i in range(len(mystring)):
	if mystring[i]=="(":
		thefloor+=1
	elif mystring[i]==")":
		thefloor-=1
	step+=1
	if thefloor==-1:
		print("Final Answer:",step)
		break

